deployment/ansible/roles/cac-tripplanner.otp-data/templates/otp.conf.j2

```diff
@@ -0,0 +1,13 @@
+## NOTICE: This file is written by ansible, and any changes made here will be overwritten on next provision.
+#           Modify azavea.opentripplanner/templates/otp.conf.j2 to make changes stick.
```

Comment:
Yeah, good idea.  Wasn't sure what would be a good way to change the template based on version, especially since we can use commits or other non-release tags, and forks.